src/main/java/mate/academy/model/User.java

```diff
@@ -5,17 +5,27 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
 import jakarta.persistence.Table;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
 import lombok.Getter;
 import lombok.Setter;
-import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
 
 @Entity
 @Getter
 @Setter
-@ToString
 @Table(name = "users")
-public class User {
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
```

Comment:
```suggestion
@SQLDelete(sql = "UPDATE users SET is_deleted = true WHERE id=?")
```
Be attentive to copy-paste